packages/office-ui-fabric-react/src/components/Button/examples/Button.Anchor.Example.tsx

```diff
@@ -24,7 +24,6 @@ export class ButtonAnchorExample extends React.Component<IButtonProps, {}> {
           href='http://bing.com'
           target='_blank'
           title='Let us bing!'
-          style={ { color: '#ffffff' } }
```

Comment:
I know this looks totally out of place, but I [added this in](https://github.com/OfficeDev/office-ui-fabric-react/pull/2920) because UHF in the website overrides our link styles. Maybe there is just a better way to go about it? Is it causing a HC issue?

Reply:
I removed it because we're changing the HC primary button style to black on white and this override made it white on white. I'm not seeing any issue with it in non-HC mode after removing this line, so I think it should be OK to remove now.

Reply:
Unfortunately the non-HC mode issue only shows up when we deploy the website. The local build doesn't have the UHF. Maybe you can test the live site in the browser to double check it's ok to remove it?

Reply:
Ah ok, that makes sense. Seems like it's still in issue. I'll come up with another way to fix this HC issue without regressing on the issue you fixed and then let you know.

Reply:
Awesome, thank you! I'll stay tuned.

Reply:
@lynamemi, I think we can sidestep this whole issue by just making this example button not primary. Let me know if you see any issue with that approach.

Also, the Persona/ActivityItem issue we saw this morning should be fixed in this PR now.

Reply:
Very good point! Let's switch that button to standard then.